contents/blog/openlineage-spark/index.mdx

```diff
@@ -43,15 +43,15 @@ some time, the recent OpenLineage 0.3 release has explicit support for Spark 3.1
 
 ## Getting Started
 
-The approach to integrating with Spark is not super novel nor is it complicated to integrate into your own system. Spark
```

Comment:
This was originally intended to communicate that the approach is not new or original- Spark users will be familiar with this mechanism, so can apply their existing understanding.